articles/azure/azs-how-create-vm.md

```diff
@@ -34,62 +34,55 @@ VMs provide the basic compute building blocks in Azure Stack. You can create VMs
 
    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
 
-2. In the favourites panel, select **New**.
+2. In the favourites panel, select **Create a resource**.
 
     ![New option in favourites panel](images/azsp_newmenu.png)
```

Comment:
Need new screen shot that shows Create a resource rather than New

Reply:
Uploaded